s2wav/axisym_filters.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from s2wav import tiling
+from s2wav import helper_functions
+from typing import Tuple
+import pytest
+
+def k_lam(L:int, lam:float, quad_iters: int=300) -> float:
+    """Compute function k_lam defined in equation (12) in [1] used to compute scaling function generating function and
+        wavelet generating function.
+
+    Args:
+        L (int): Harmonic band-limit.
+        lam (float): Wavelet parameter which determines the scale factor between consecutive wavelet scales.
+        quad_iters (int): Total number of iterations for quadrature integration. Defaults to 300.
+    Returns:
+        np.ndarray: value of k_lam computed for values between 1/lam and 1, parametrised by l as required to compute 
```

Comment:
```suggestion
        (np.ndarray): value of k_lam computed for values between 1/lam and 1, parametrised by l as required to compute 
```

Reply:
Think the data type needs to be in brackets for it to be recognized correctly in the google doc style.

Reply:
I'm not 100% certain this is mandatory, but we should be consistent throughout the code, and this is the convention used elsewhere :) I've updated appropriately.